src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Given a tweet or sentence, the servlet will extract salient terms
+ * and store them in a list.
+ */
+@WebServlet("/keyword")
+public class KeywordServlet extends HttpServlet {
+
+  private static final List<String> tweets = new ArrayList<>();
+  private static final List<String> terms = Arrays.asList("Black Lives Matter", "COVID-19");
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Extract saliency.
+    String json = jsonBuilder(terms);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String tweet = request.getParameter("keyword-sentence");
+    if (tweet != null && !tweet.equals("")) {
+      tweets.add(request.getParameter("keyword-sentence"));
+    }
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * Generates and returns a JSON usable by Mustache.
+   */
+  private String jsonBuilder(Collection<String> toJson) {
+    List<String> terms = new ArrayList<>();
+    for (String term : toJson) {
+      terms.add(keyValueJson("term", term));
+    }
+
+    // Remove extra comma at the end.
+    String keywordPrefix = "{\"keywords\": [";
+    return String.format(keywordPrefix + "%s]}", String.join(",", terms));
```

Comment:
[Nit] I include the keywordPrefix variable in my suggestion but then realized it wasn't necessary because it is only used one.  I would just hard code it here.

Reply:
Also, I think you can use your helper function in the return here now.

`return keyValueJson("keywords", String.format("[%s]", String.join(",", terms))`

I'm still partial to using an actual JSON library for this, but you can do that in a future PR if you want.

Reply:
Yea, I realized that and removed the prefix variable, but it's not pushed yet (Might be something you see in the Saliency branch as Nathan mentioned something similar to this there.

As for the JSON library, I will probably leave that for a future PR.

Reply:
Ok, I'll approve this PR as soon the changes to remove keywordPrefix variable are committed (I'm assuming that change is going to be made to this PR)